README.md

```diff
@@ -33,6 +33,33 @@ tabrmd on the system bus as this is the most common configuration.
 
```

Comment:
There's a typo in the commit message s/LCP/LPC. And also I would say change LPC this year to LPC 2017. It saves you one level of indirection on having to look at the commit date to know what year that is.

Reply:
I think I'm physically incapable of typing LPC given the number of times I've typed out LCP (launch control policy)